### PR TITLE
Add project prometheus node_exporter

### PR DESCRIPTION
**Pull request by ivyostosh**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/prometheus/node_exporter/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+BINARY_TARGET_FILES=node_exporter
+EXTRA_GOBUILD_FLAGS=-tags netgo,osusergo,static_build
+
+NODE_EXPORTER_IMAGE_COMPONENT=prometheus/node-exporter
+
+EXCLUDE_FROM_STAGING_BUILDSPEC=true
+
+include $(BASE_DIRECTORY)/Common.mk
+
```
Comment (TerryHowe) [inline projects/prometheus/node_exporter/Makefile:16]: on the topic of minimal base images, seems like this could work with one

**Comment by TerryHowe** [comment]
conflict

File: projects/prometheus/node_exporter/Makefile (hunk 1 above)
Comment (ivyostosh) [reply projects/prometheus/node_exporter/Makefile:17]: Updated

**Review by ivyostosh — COMMENTED** [review]
Conflict resolved as well

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**